src/MainWindow.vala

```diff
@@ -150,6 +150,7 @@ namespace Terminal {
             action_accelerators[ACTION_NEW_WINDOW] = "<Control><Shift>n";
             action_accelerators[ACTION_NEXT_TAB] = "<Control><Shift>Right";
             action_accelerators[ACTION_PREVIOUS_TAB] = "<Control><Shift>Left";
+            action_accelerators[ACTION_PREVIOUS_TAB] = "<Ctrl>Tab";
```

Comment:
Shouldn't `<Ctrl>Tab` navigate to *next* tab?  And `<Shift><Ctrl>Tab` navigate to the previous tab?  Problem is, Vte-91 captures `Tab` && `<Ctrl>Tab` for triggering autocompletion.

Reply:
Hi there,

1. I will change to *next tab* 
1. here it works nice and don't trig autocomplete 

![Peek 2020-08-25 11-14](https://user-images.githubusercontent.com/1039615/91185498-498a4b80-e6c4-11ea-9e98-eaaa1eb76353.gif)

Reply:
Hmm, you are right - cloning your branch it works - previously I just amended my own repo but I used "<Control>Tab" like the rest of the code.  Not sure why that should make a difference.

Reply:
> `<Ctrl>Tab` does not _seem_ to do anything different from `Tab` in Vte.Terminal, but I am not sure.

fix:

- [ x ] using `<Control>`
- [ x ] `<Control>Tab`  for next tab
- [ x ] `<Control><Shift>Tab` for previous tab